ui/app/AppMain.qml

```diff
@@ -170,6 +174,12 @@ Item {
         }
     }
 
+    property Component changeProfilePicComponent: Component {
```

Comment:
Does it need to be property? See `downloadModalComponent` as an example.

Reply:
@osmaczko great point. It doesn't need to be a property, but it could be. There is no a big difference, it's more a matter of coding style used in the app. But since you most likely was about that we should be uniform in regards of that across the app, I fully agree. How I actually ended up here with the property instead of component, cause I just copied a component above it (`profilePopupComponent` one in the line 164).

Reply:
That's right, there is no big difference. One subtle difference is that property can be reassigned and so the component can be changed at runtime. It could be especially useful if `AppMain` is a reusable component and it's up to the client how `changeProfilePicComponent` should/could look like.. but it's not the case here.